app/models/reaction.rb

```diff
@@ -82,15 +82,15 @@ def cached_any_reactions_for?(reactable, user, category)
     # @return [TrueClass] yup, they're spamming the system.
     # @return [FalseClass] they're not (yet) spamming the system
     def user_has_been_given_too_many_spammy_article_reactions?(user:, threshold: 2)
-      article_vomits.where(reactable_type: "Article", reactable_id: user.articles.ids).size > threshold
+      article_vomits.where(reactable_id: user.articles.ids).size > threshold
```

Comment:
github hid the scopes in the default diff view (line 36 above) - but this sure looks like it was redundant (though I would be surprised if it was incorrect?)

```
  scope :article_vomits, -> { where(category: "vomit", reactable_type: "Article") }
  scope :comment_vomits, -> { where(category: "vomit", reactable_type: "Comment") }
````